controller/api/v1alpha1/exporter_helpers_test.go

```
@@ -84,6 +84,8 @@ func TestExporter_Usernames(t *testing.T) {
 }
 
 // Helper function to create string pointers
+//
+//go:fix inline
 func stringPtr(s string) *string {
```

golangci-lint (2.13.2)

[error] func stringPtr is unused

(unused)